commit 460bad5fe1ae2ebcd9cb73ad7b51538996b722b2
Author: Boris Brezillon <boris.brezillon@free-electrons.com>
Date:   Thu Oct 5 18:53:19 2017 +0200

    mtd: nand: Export nand_reset() symbol
    
    commit b9bb98424c51437973b854691aa1e9b2bfd348f5 upstream.
    
    Commit 6e532afaca8e ("mtd: nand: atmel: Add PM ops") started to use the
    nand_reset() function which was not yet exported by the NAND framework
    (because it was only used internally before that). Export this symbol
    to avoid build errors when the driver is enabled as a module.
    
    Fixes: 6e532afaca8e ("mtd: nand: atmel: Add PM ops")
    Signed-off-by: Boris Brezillon <boris.brezillon@free-electrons.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/mtd/nand/nand_base.c b/drivers/mtd/nand/nand_base.c
index 12edaae17d81..fa451c1f30b3 100644
--- a/drivers/mtd/nand/nand_base.c
+++ b/drivers/mtd/nand/nand_base.c
@@ -1246,6 +1246,7 @@ int nand_reset(struct nand_chip *chip, int chipnr)
 
 	return 0;
 }
+EXPORT_SYMBOL_GPL(nand_reset);
 
 /**
  * nand_check_erased_buf - check if a buffer contains (almost) only 0xff data
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1119_460bad5fe1ae2ebcd9cb73ad7b51538996b722b2.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1119_460bad5fe1ae2ebcd9cb73ad7b51538996b722b2.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
